operate/wallet/master.py

```diff
@@ -100,10 +102,15 @@ def ledger_api(
         rpc: t.Optional[str] = None,
     ) -> LedgerApi:
         """Get ledger api object."""
+        gas_price_strategies = deepcopy(DEFAULT_GAS_PRICE_STRATEGIES)
+        if chain == Chain.BASE:
+            gas_price_strategies[EIP1559]["fallback_estimate"]["maxFeePerGas"] = to_wei(5, GWEI)
+
```

Comment:
Interesting but IMO these kind of issues should be handled by the core library. An issue should be opened.

Reply:
Opened [here](https://github.com/valory-xyz/open-aea/issues/775)